pyt/__main__.py

```diff
@@ -235,7 +238,28 @@ def main(command_line_args=sys.argv[1:]):
     elif args.trim_reassigned_in:
         ui_mode = UImode.TRIM
 
+    path = os.path.normpath(args.filepath) 
     cfg_list = list()
+    if args.ignore_nosec:
```

Comment:
i moved the code this place and deleted `nosec_lines = set() ` .Because its already created in if-else statement.